packages/table/components/Table.tsx

```diff
@@ -55,20 +63,24 @@ export const fillColumns = (
   width: number
 ): number[] => {
   let copiedColWidths = [...colWidths];
-  const widthToFill =
-    width - copiedColWidths.reduce((acc, curr) => acc + curr, 0);
+  const filledArea = columns
+    .filter(child => !child.props.growToFill)
+    .map(col => copiedColWidths[columns.indexOf(col)])
+    .reduce((acc, curr) => acc + curr, 0);
+  const widthToFill = width - filledArea;
   const fillWidthColsIdxArr: number[] = columns
     .filter(child => child.props.growToFill)
     .map(col => columns.indexOf(col));
 
   fillWidthColsIdxArr.forEach(colIdx => {
     const oldWidth = copiedColWidths[colIdx];
-    const newWidth = widthToFill / fillWidthColsIdxArr.length + oldWidth;
+    const newWidth = widthToFill / fillWidthColsIdxArr.length;
```

Comment:
I made some changes to `fillColumnWidth` to fix a bug where after deleting a row, if you hovered over the table or triggered a re-render, the column widths would "jump". I found this while testing Georgi's requested changes.

Before:
![coljump_before](https://user-images.githubusercontent.com/2313998/52366453-c76dff80-2a17-11e9-81fb-6a2c27b5ffb6.gif)

After:
![coljump_after](https://user-images.githubusercontent.com/2313998/52366550-069c5080-2a18-11e9-97e0-dd28ea736005.gif)